doc/moveit_cpp/config/controllers.yaml

```diff
@@ -0,0 +1,15 @@
+controller_names:
```

Comment:
We should use the one from [moveit_resources](https://github.com/ros-planning/moveit_resources/blob/ros2/panda_moveit_config/config/panda_controllers.yaml)